tests/DriverManagerTest.php

```diff
@@ -233,6 +233,47 @@ public function databaseUrls(): iterable
                     'driver' => PDO\SQLite\Driver::class,
                 ],
             ],
+            'sqlite relative URL without host and spetial characters in path' => [
+                'sqlite:///foo/u~z/#f`aa/*  *o/dbna] me.sqlite',
```

Comment:
This is an invalid URL. The DBAL should not implement parsing invalid URLs.

Reply:
Thank you for your review, i improve it